bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -4138,6 +4420,9 @@ ReplicaImp::ReplicaImp(const LoadedReplicaData &ld,
       }
 
       if (e.getForceCompleted()) seqNumInfo.forceComplete();
+      if (e.getIsExecuted()) ConcordAssert(e.getPrePrepareMsg()->seqNumber() > lastExecutedSeqNum);
```

Comment:
NIT: I think it should be isExecuted and not getIsExecuted

Reply:
I followed the convention in `PersistentStorageWindows.hpp` where a getter name starts with `get`